src/enums/associationTypes.ts

```diff
@@ -1,6 +0,0 @@
-/** Association types - only used for COOP for now. */
-export enum AssociationTypes {
-    COMMUNITY_SERVICE_COOPERATIVE = 'CSC',
-    HOUSING_COOPERATIVE = 'HC',
-    ORDINARY_COOPERATIVE = 'CP',
-}
```

Comment:
Moved to entitySubtypes.ts (same as Create UI).

Ditto for BusinessTypes below.